src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerBuildChooser.java

```diff
@@ -182,7 +182,7 @@ public ObjectId invoke(Repository repository, VirtualChannel virtualChannel)
                     throw new GitException("Failed to find parent id. ", e);
                 } finally {
                     if (walk != null) {
-                        walk.release();
+                        walk.close();
```

Comment:
Since it now is a closeable and java 7; I think we should use the "try with resource" form instead.